src/components/maincontroller.ts

```diff
@@ -48,6 +48,9 @@ let broadcastToServer = new Date();
 
 let hasReportedCapabilities = false;
 
+/**
+ *
+ */
```

Comment:
jsdoc needs to be completed, same for all the following ones.

Reply:
There should now be a lint rule to warn on empty blocks as well, so this should still produce a warning